src/main/java/net/mcreator/ui/workspace/WorkspacePanel.java

```diff
@@ -771,12 +771,13 @@ public WorkspacePanel(final MCreator mcreator) {
 
 		addVerticalTab("mods", L10N.t("workspace.category.mod_elements"),
 				new WorkspacePanelMods(PanelUtils.westAndCenterElement(toolp, modElementsPanel)));
-		verticalTabs.get(0).doClick();
 		addVerticalTab("res", L10N.t("workspace.category.resources"), resourcesPan);
 		addVerticalTab("locales", L10N.t("workspace.category.variables"), new WorkspacePanelVariables(this));
 		addVerticalTab("variables", L10N.t("workspace.category.localization"), new WorkspacePanelLocalizations(this));
 		addVerticalTab("vcs", L10N.t("workspace.category.remote_workspace"), new WorkspacePanelVCS(this));
 
+		verticalTabs.get(0).doClick();

```

Comment:
This does the same thing as
https://github.com/MCreator/MCreator/blob/30a5e9769ba126aa6af11d5c1a73d93d26fc91ea/src/main/java/net/mcreator/ui/workspace/WorkspacePanel.java#L99

Reply:
It is there a sanity check as some methods use currentTab and could cause NPE if someone called one of those too early